go/service/pgp.go

```diff
@@ -162,7 +162,7 @@ func (h *PGPHandler) PGPImport(_ context.Context, arg keybase1.PGPImportArg) err
 		LogUI:     h.getLogUI(arg.SessionID),
 		SessionID: arg.SessionID,
 	}
-	eng, err := engine.NewPGPKeyImportEngineFromBytes(arg.Key, arg.PushSecret, nil)
+	eng, err := engine.NewPGPKeyImportEngineFromBytes(arg.Key, arg.PushSecret, h.G())
```

Comment:
Well, libkb.Warning doesn't use contextified:

https://github.com/keybase/client/blob/master/go/libkb/warning.go

and there's this:

https://github.com/keybase/client/blob/master/go/engine/pgp_import_key.go#L58